src/main/java/racingcar/InputView.java

```diff
@@ -0,0 +1,28 @@
+package racingcar;
+
+import java.util.Scanner;
+
+public class InputView {
+    private final static Scanner scanner = new Scanner(System.in);
```

Comment:
접근 제어자 순서 [컨벤션](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers)과 상수명 [컨벤션](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names) 확인 부탁드려요 
```suggestion
public class InputView {
    private static final Scanner SCANNER = new Scanner(System.in);
```